product.json

```diff
@@ -5,7 +5,7 @@
 	"dataFolderName": ".azuredatastudio",
 	"win32MutexName": "azuredatastudio",
 	"licenseName": "Microsoft EULA",
-	"licenseUrl": "https://raw.githubusercontent.com/microsoft/azuredatastudio/main/LICENSE.txt",
+	"licenseUrl": "https://learn.microsoft.com/azure-data-studio/license",
 	"serverLicenseUrl": "https://raw.githubusercontent.com/microsoft/azuredatastudio/main/LICENSE.txt",
```

Comment:
when I searched for `serverLicenseUrl` it looked like it was used for web mode, so I didn't update this link. Let me know if it should be updated as well